pkgs/tools/security/eid-mw/default.nix

```
@@ -8,29 +8,24 @@
 
 stdenv.mkDerivation rec {
   pname = "eid-mw";
-  version = "4.4.16";
+  version = "4.4.27";
 
-  src = fetchFromGitHub {
-    sha256 = "1q82fw63xzrnrgh1wyh457hal6vfdl6swqfq7l6kviywiwlzx7kd"; 
-    rev = "v${version}";
-    repo = "eid-mw";
-    owner = "Fedict";
+  src = fetchurl {
+    url = "https://dist.eid.belgium.be/continuous/sources/eid-mw-${version}-v${version}.tar.gz";
+    sha256 = "02dbwgnkzpxkc4avs7zcrrwbjn2v59g5ffpb6vcfi80ybimw9hm6";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig makeWrapper ];
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
   buildInputs = [ gtk3 pcsclite libxml2 libproxy curl openssl ];
   preConfigure = ''
     mkdir openssl
     ln -s ${openssl.out}/lib openssl
     ln -s ${openssl.bin}/bin openssl
     ln -s ${openssl.dev}/include openssl
     export SSL_PREFIX=$(realpath openssl)
+    sed -i "s%c_rehash%openssl rehash%g" plugins_tools/eid-viewer/Makefile.in
```

For simple substitutions that do not rely on regular expressions, `substituteInPlace` is more canonical:

```suggestion
    substituteInPlace plugins_tools/eid-viewer/Makefile.in \
      --replace "c_rehash" "openssl rehash"
```